### ISSUE TITLE
[BUG]: Shin Megami Tensei - Persona 3 (SLUS-21569) FMVs flickering.

### ISSUE THREAD

**Issue by RVRZERO** [PCSX2/pcsx2#11575]
### Describe the Bug

All FMVs flickers.

### Reproduction Steps

Just launch the game with PCSX2 default settings.

### Expected Behavior

It shouldn't flicker like that, in Persona 3 FES which is essentially the same game, this issue doesn't happen.

I actually found the root of the problem. It is from the auto game fixes that are applied from GameIndex.yaml, specifically a SoftwareRendererFMVHack that is supposed to fix interlacing which it does but the FMVs flickers so hard in return.

I would appreciate it if you could fix it so it doesn't flicker either by removing the hack completely or by fixing it, it is up to you.

### PCSX2 Revision

v2.0.3

### Operating System

Windows 10 (64bit)

### CPU

i7-7700k

### GPU

GTX 1080TI

### Logs & Dumps

[Uploading Shin Megami Tensei - Persona 3_SLUS-21569_FMVs flickering.zip…]()

**Comment by RVRZERO** [comment]
GS Dumps didn't upload correctly, here is a re upload.

[Shin Megami Tensei - Persona 3_SLUS-21569_FMVs flickering.zip](https://github.com/user-attachments/files/16267854/Shin.Megami.Tensei.-.Persona.3_SLUS-21569_FMVs.flickering.zip)

**Comment by Mrlinkwii** [comment]
What's the status of this on latest nightly?

**Comment by RVRZERO** [comment]
> What's the status of this on latest nightly?

Still the same result in the latest nightly (2.3.261).

GS Dump: 

[Shin Megami Tensei - Persona 3_SLUS-21569_20250409091117.gs.zip](https://github.com/user-attachments/files/19661490/Shin.Megami.Tensei.-.Persona.3_SLUS-21569_20250409091117.gs.zip)

**Comment by Mrlinkwii** [comment]
What's the status of this on latest nightly?

**Comment by refractionpcsx2** [comment]
I have more hope in this after #13701 merges (but I've not checked)

**Comment by RVRZERO** [comment]
> What's the status of this on latest nightly?

Tested on 2.7.38
It doesn't flicker anymore but it does have tearing, with and without SoftwareRendererFMVHack in the GameIndex.yaml. Although to be clear I fully changed my hardware recently so here is the GS Dump If you can check it yourself it would be great to make sure it is not related to my monitor or something since i have g-sync enabled.

[Shin Megami Tensei - Persona 3_SLUS-21569_20260115021000.gs.zip](https://github.com/user-attachments/files/24627301/Shin.Megami.Tensei.-.Persona.3_SLUS-21569_20260115021000.gs.zip)

**Comment by AmandaRoseChaqueta** [comment]
> > What's the status of this on latest nightly?
> 
> Tested on 2.7.38 It doesn't flicker anymore but it does have tearing, with and without SoftwareRendererFMVHack in the GameIndex.yaml. Although to be clear I fully changed my hardware recently so here is the GS Dump If you can check it yourself it would be great to make sure it is not related to my monitor or something since i have g-sync enabled.
> 
> [Shin Megami Tensei - Persona 3_SLUS-21569_20260115021000.gs.zip](https://github.com/user-attachments/files/24627301/Shin.Megami.Tensei.-.Persona.3_SLUS-21569_20260115021000.gs.zip)

I don't see any tearing or flickering

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/2a650856-241c-47e7-9849-cad8af0fc83e" />

**Comment by AmandaRoseChaqueta** [comment]
@RVRZERO Can you test again? Cuz On linux using software or hardware I don't see any flickering or tearing. Others might wanna test this too

**Comment by RVRZERO** [comment]
> [@RVRZERO](https://github.com/RVRZERO) Can you test again? Cuz On linux using software or hardware I don't see any flickering or tearing. Others might wanna test this too

Tested again on v2.7.141 with Default settings.

The flickering issue is finally fixed but another issue appeared.

The new issue is a green line flickering below in some of the FMVs and it also happen in software mode.

Here is a screenshot with the green line below

<img width="3840" height="2088" alt="Image" src="https://github.com/user-attachments/assets/9df14b61-754f-4829-aa77-24a1748b27b1" />

For some reason it doesn't appear in the GS Dump because the GS Dump seems to not have the SoftwareRendererFMVHack applied at all because i can observe interlacing effects and the green line doesn't appear.

GS Dump: 

[Shin Megami Tensei - Persona 3_SLUS-21569_20260225060929.gs.zip](https://github.com/user-attachments/files/25535356/Shin.Megami.Tensei.-.Persona.3_SLUS-21569_20260225060929.gs.zip)

Additional note: In software mode, interlacing effect is visible which means that SoftwareRendererFMVHack is better at deinterlacing the FMVs of this game than software mode if we ignore the green line issue.

**Comment by AmandaRoseChaqueta** [comment]
@RVRZERO Could you provide a savestate just before that cutscene?

**Comment by Mrlinkwii** [comment]
Can this rechecked on latest  nightly
